content/_en/partners/index.md

```diff
@@ -5,37 +5,84 @@ redirect_from:
 - /partners/learn/
 - /partners/our-agency-partners/
 - /partners/why-login-gov/
-title: Login.gov Partners
-description: Learn how Login.gov can better provide identity proofing for your agency.
 title: Simple, secure online access for the public
 description: Login.gov adds the public’s one account for government to your agency.
 class: partners
-three_col:
-  heading: Our Services
+
+services:
+  heading: Our services
   subheading1: Authentication
   col_class1: authentication
   col1: >-
     Users create one account to access all your digital services with multi-factor authentication.
 
-    [Learn about authentication](https://login.gov/partners){:class="partners-authentication"}
```

Comment:
These links just go to /partners

Reply:
You're right! Looks like they should go to `Our Services`.